src/scripts/main.js

```diff
@@ -1,5 +1,38 @@
 'use strict';
 
+const spiderElement = document.querySelector('.spider');
+const wallElement = document.querySelector('.wall');
+
 document.addEventListener('click', (e) => {
   // write code here
+
+  if (!wallElement.contains(e.target)) {
+    return;
+  }
+
+  const wallBorderSize = parseInt(getComputedStyle(wallElement).borderWidth);
```

Comment:
The use of `parseInt` to get the border width is appropriate here, as `getComputedStyle` returns a string with 'px' suffix. This ensures the border width is correctly subtracted from the calculated positions.